Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Runs master branch from github of lando/lando-messaging
+# Requires a lando config file mounted at /etc/lando_config.yml.
+FROM python:2.7.13
+RUN pip install git+git://github.com/Duke-GCB/lando-messaging.git
+RUN pip install git+git://github.com/Duke-GCB/lando.git
```

Comment:
Since the Dockerfile is in the repo, wouldn't it make more sense to install from the local source, e.g.:

```
ADD . /src
WORKDIR /src
RUN python setup.py install
```

Otherwise, the Dockerfile would never build an alternate branch or local changes.